public/compute/first_contact.md

```diff
@@ -24,41 +86,84 @@ In Molgenis Compute, anything can be a parameter.
 You can specify parameter values at generation time in parameter files.
 In fact, for some parameters you may want to specify multiple values for some parameters.
 
-## Trying it out
-### Create a workflow
-We assume you have downloaded and unzipped molgenis compute commandline and are now in the directory containing the unzipped files.
+In this workflow, the parameter values are listed in two files:
 
-You can generate a template for a new workflow using the command:
+### parameters.csv
+|input|
+|-----|
+|hello|
+|bye  |
 
-```bash
-  sh molgenis_compute.sh --create myfirst_workflow
+The `input` parameter has two different values, hello and bye.
+In `step1`, the `input` parameter gets mapped to the `in` parameter of `protocols/step1.sh`.
+The parameter file lists two values for this parameter so this script will be created twice, one version for `input=hello`, and one for `input=bye`
+
+### workflow.defaults.csv
+|workflowName|creationDate|
+|------------|------------|
+|myFirstWorkflow|today    |
+
+These parameter values will be inserted into the template for `step2`.
+Let's take a look at `protocols/step2.sh`
+
+```FreeMarker
+#string wf
+#string date
+#list strings
+
+echo "Workflow name: ${wf}"
+echo "Created: ${date}"
+
+echo "Result of step1.sh:"
+for s in "${strings[@]}"
+do
+    echo ${s}
+done
+
+
+echo "(FOR TESTING PURPOSES: your runid is ${runid})"
 ```
 
-This will create a new directory for the workflow:
+The first three lines are the header listing the input parameters of the protocol.
+In `step2` maps the parameter value `workflowName=myFirstWorkflow` from file `workflow.defaults.csv` to protocol input `wf`. So when the script for `step2` gets generated, the value `myFirstWorkflow` will be filled in where it says `${wf}`.
```

Comment:
In `step2` the parameter value `workflowName=myFirstWorkflow` from file `workflow.defaults.csv` is mapped to protocol input `wf`